src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,8 +50,18 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+//            if (!Collections.disjoint(wordsInName, keywords)) {
+//                matchedPersons.add(person);
+//            }
```

Comment:
Try not to commit commented-out code like this. Git keeps a record of code changes, if you really need it back in the future, you can just revert.